portal/src/graphql/adminapi/AddGroupScreen.tsx

```diff
@@ -188,6 +188,11 @@ const AddGroupScreen: React.VFC = function AddGroupScreen() {
     validate,
   });
 
+  const canSave = useMemo(
+    () => form.state.groupName !== "",
```

Comment:
I think we need to check groupKey too, same as AddRoleScreen?

Reply:
groupKey is auto filled when it's empty